lib/client.js

```diff
@@ -72,24 +124,29 @@ bedrock.events.on('bedrock-mongodb.ready', function(callback) {
  * @param interval the number of minutes to wait between polling
  * the endpoint (e.g. 1, 5, or 10)
  */
-api.start = function(endpoint, interval, callback) {
-  // TODO: Where validation of the interval being equal to 1,5, or 10 be located
-  var jobId = createJobId(endpoint);
+api.start = function(client, callback) {
+  if(!client.interval || !client.id) {
+    return callback(new BedrockError(
+      'Could not start the client.',
+      'SchedulingFailure',
```

Comment:
This should be throwing an TypeError because it was called improperly, rather than passing an error to the callback because of some exceptional condition that arose. Something like this:

```js
throw new TypeError('"client.id" and "client.interval" must be strings.');
```